packages/terra-theme-context/src/ThemeContext.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ThemeProviderContext = React.createContext({});
+
+const contextShape = PropTypes.shape({
+  /**
+   * The current application theme className.
+   * The default theme is indicated as undefined.
+   */
+  className: PropTypes.string,
+});
+
+export default ThemeProviderContext;
+export { contextShape };
```

Comment:
In orion-application these usually have a default name scope. `{ themeContextShape }`

Reply:
i based it on this one: https://github.com/cerner/terra-application/blob/master/src/application-loading-overlay/ApplicationLoadingOverlayContext.jsx#L23

Reply:
no harm in being more specific though